.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+### Status
+
+Work in Progress / Ready for review
+
+### Review Checklist
+
+- [ ] Changes to `onboarded.txt` are accurate
+- [ ] The file `default.rulesets.TIMESTAMP.gz` has been updated, extracting that file and inspecting the contents of the JSON file produces the expected rules
+- [ ] The ruleset has been verified by modifying the httpseverywhere configuration in a Tor Browser instance pointin to `Path Prefix`: `https://raw.githubusercontent.com/freedomofpress/securedrop-https-everywhere-ruleset/$BRANCH_NAME`
```

Comment:
The suggestion for using the feature branch name for functional testing surprised me, since GitHub doesn't do a good job of serving up lfs assets, at least not as part of GH Pages. Turns out we're not using LFS on this repo, however, and the binary tarballs are committed as blobs directly to the repo. Something to revisit in the near future, perhaps.

Reply:
typo: pointin -> pointing,  fixin'

re: use of LFS, the long term goal will be to generate these rules directly from the SD directory I think, that seems likea good time to decide if we want it to be backed by LFS or skip using a git repo altogether

Reply:
Done in b4fc1d1c521c38a121b7758c3a275f7d5065bdce

Reply:
> the long term goal will be to generate these rules directly from the SD directory I think

:+1: